crates/nu-cli/src/app.rs

```diff
@@ -162,12 +236,38 @@ impl App {
             return Ok(());
         }
 
+        if self.perf() {
+            // start a new split
+            eprintln!(
+                "run_script_file commands 2: {:?}",
```

Comment:
This could be `run script file(s):`

Reply:
so this is like `nu test.nu` i guess? i'll take your suggestion

Reply:
I believe so